Makefile

```diff
@@ -912,16 +912,21 @@ GENERATOR_BUILD_RUNGEN_TESTS := $(filter-out $(FILTERS_DIR)/old_buffer_t.rungen,
 GENERATOR_BUILD_RUNGEN_TESTS := $(filter-out $(FILTERS_DIR)/tiled_blur.rungen,$(GENERATOR_BUILD_RUNGEN_TESTS))
 test_rungen: $(GENERATOR_BUILD_RUNGEN_TESTS)
 
-test_generators: $(GENERATOR_AOT_TESTS) $(GENERATOR_AOTCPP_TESTS) $(GENERATOR_JIT_TESTS) $(GENERATOR_BUILD_RUNGEN_TESTS)
+test_generator: $(GENERATOR_AOT_TESTS) $(GENERATOR_AOTCPP_TESTS) $(GENERATOR_JIT_TESTS) $(GENERATOR_BUILD_RUNGEN_TESTS)
 
-ALL_TESTS = test_internal test_correctness test_errors test_tutorials test_warnings test_generators
+# TODO: this is a temporary target added to allow existing buildbot to run without breaking;
+# it will be removed after buildbot is updated.
+.PHONY: test_generators
+test_generators: test_generator
+
+ALL_TESTS = test_internal test_correctness test_errors test_tutorials test_warnings test_generator
```

Comment:
Seems like test_errors, test_tutorials, test_warnings have the same problem?

Reply:
Good point. Might as well nail them all at once.